pkgs/top-level/all-packages.nix

```diff
@@ -4802,6 +4802,10 @@ with pkgs;
 
   slsnif = callPackage ../tools/misc/slsnif { };
 
+  slstatus = callPackage ../applications/misc/slstatus {
+    conf = config.slstatus.conf or null;
```

Comment:
I like this idea, and I see `config` being used in other places throughout nixpkgs, but I'm not sure whether I understand the purpose, because it's just as easy to use `slstatus.override { configFile = ./foobar.def.h; }` instead